pom.xml

```diff
@@ -4,8 +4,8 @@
 
     <parent>
         <groupId>guru.nidi</groupId>
-        <artifactId>guru-nidi-parent-pom</artifactId>
-        <version>1.0.5</version>
+        <artifactId>guru-nidi-parent-pom-jdk6</artifactId>
```

Comment:
Indeed, I only changed the java.version property from the parent pom. I corrected this by reverting to the original parent pom version and by overriding the compiler plugin to set the java version to 1.6.
I also changed the artifactId to include a jdk6 qualifier.